Source/EVEMarketeerTest/EVEMarketeerTest/JSONTester.cs

```diff
@@ -0,0 +1,28 @@
+﻿using EVEClassLibrary;
+using System.Net;
+
+namespace EVEMarketeerTest
+{
+
+    [TestClass]
+    public class JSONTester 
```

Comment:
Rename class to what it tests + "tests".

Rename the following:
Class "JSONTester" to "JSONRipperTests"
Function "JSONFormatTest" to "Formatter_working"